### ISSUE TITLE
Homebridge v2.0 Compatibility

### ISSUE THREAD

**Issue by badgertastic** [rnilssoncx/homebridge-pico#55]
Hi, Any plans to test homebridge-pico for compatibility with Homebridge v2 and update if needed? Thanks

**Comment by rnilssoncx** [comment]
Yes. I plan to review and make any changes this month.

**Comment by Fofer** [comment]
Yes please, I love this plugin so much but now that Homebridge 2.0 is publicly released this is the only one left preventing me from upgrading :(

**Comment by smedsters** [comment]
> Yes. I plan to review and make any changes this month.

Has the compatibility with 2.0 been verified?  Thanks.
